src/components/common/TmSelectNetwork.vue

```diff
@@ -46,9 +46,10 @@ export default {
     sortedNetworks() {
       // sorts networks setting showing the current network first, mainnets at the top and the default one the first
       if (this.networks.length > 0) {
+        const selectedNetwork = this.networks.find(({ id }) => id === this.networkId)
         return [
           // current network first
-          this.networks.find(({ id }) => id === this.networkId),
+          ...(selectedNetwork ? [selectedNetwork] : []),
```

Comment:
why this complicated syntax?

Reply:
We can use the extension without the networkId set. And in this case we are getting an error in previous code.

Reply:
instead why not use `.filter(x => !!x)` to filter out undefineds?

Reply:
I will create a PR for that

Reply:
https://github.com/luniehq/lunie/pull/3493